app/javascript/src/components/actions/submittable.js

```diff
@@ -0,0 +1,26 @@
+
+export function submittable(node) {
+  const nodeIsInput = node.tagName === "INPUT"
+  const nodeIsTextarea = node.tagName === "TEXTAREA"
+
+  if (!(nodeIsInput || nodeIsTextarea)) {
+    throw new Error("submittable action is only meant to be used on inputs")
+  }
+
+  function keydown(event) {
+    if (event.code !== "Enter") return
+    if (nodeIsTextarea && (event.metaKey || event.altKey || event.shiftKey || event.ctrlKey)) return
```

Comment:
I think this should happen for all inputs, not just textarea. If you are holding a modifier key your intent is most likely _not_ to submit

```suggestion
  function keydown(event) {
    if (event.code !== "Enter") return
    if (event.metaKey || event.altKey || event.shiftKey || event.ctrlKey) return
```